src/main/scala/nl/vroste/zio/kinesis/client/DynamicConsumer.scala

```diff
@@ -50,7 +51,8 @@ object DynamicConsumer {
     cloudWatchClientBuilder: CloudWatchAsyncClientBuilder = CloudWatchAsyncClient.builder,
     dynamoDbClientBuilder: DynamoDbAsyncClientBuilder = DynamoDbAsyncClient.builder(),
     initialPosition: InitialPositionInStreamExtended =
-      InitialPositionInStreamExtended.newInitialPosition(InitialPositionInStream.TRIM_HORIZON)
+      InitialPositionInStreamExtended.newInitialPosition(InitialPositionInStream.TRIM_HORIZON),
+    isEnhancedFanOut: Boolean = false
```

Comment:
Would a default true make more sense you think? Or are there downsides to using this by default?

Reply:
You are right - a default of `true` would make more sense going forward - I just did not want to break existing code. If you are not worried with breaking changes I will change this?

Reply:
We can consider the default of not having enhanced fan out as a bug.